returnn/config.py

```diff
@@ -131,6 +131,58 @@ def get(self, key, default=None):
             return self.post_config[key]
         return self.config.get(key, default)
 
+    def update(self, other):
+        """
+        updates a ReturnnConfig with an other ReturnnConfig
+
+        :param ReturnnConfig other:
+        """
+        self.config.update(other.config)
+        self.post_config.update(other.post_config)
+        if other.staged_network_dict is not None:
+            self.staged_network_dict = other.staged_network_dict
+
+        if self.python_prolog is None:
+            self.python_prolog = other.python_prolog
+        elif isinstance(self.python_prolog, "str") and isinstance(
+            other.python_prolog, "str"
+        ):
+            self.python_prolog += other.python_prolog
+        else:
+            self.python_prolog = [self.python_prolog, other.python_prolog]
+        if self.python_prolog_hash is None:
+            self.python_prolog_hash = other.python_prolog_hash
+        if isinstance(self.python_prolog_hash, "str") and isinstance(
+            other.python_prolog_hash, "str"
+        ):
+            self.python_prolog_hash += other.python_prolog_hash
+        else:
+            self.python_prolog_hash = [
+                self.python_prolog_hash,
+                other.python_prolog_hash,
+            ]
+
+        if isinstance(self.python_epilog, "str") and isinstance(
```

Comment:
handling of `python_prolog` and `python_epilog` should be the same, right?

but code is different.

could this be moved to a function??

Reply:
> handling of python_prolog and python_epilog should be the same, right?

unfortunately the default argument of prolog is `None` while epilog is `""`. therefore the different handling.
https://github.com/rwth-i6/i6_core/blob/48ec2dfd3d2dde9305892428f16c91f585d4aea8/returnn/config.py#L78-L81